website/templates/include/comment_template.mako

```diff
@@ -69,12 +69,12 @@
                         -->
                         <div data-bind="template {if: editing, afterRender: autosizeText}">
                             <div class="form-group" style="padding-top: 10px">
-                                <textarea class="form-control" data-bind="value: content, valueUpdate: 'input', attr: {maxlength: $root.MAXLENGTH}"></textarea>
+                                <div class="form-control atwho-input" placeholder="Add a comment" data-bind="editableHTML: content, attr: {maxlength: $root.MAXLENGTH}" contenteditable="true"></div>
```

Comment:
The placeholder should be "Edit comment" in this case.

![screen shot 2016-04-19 at 6 33 26 pm](https://cloud.githubusercontent.com/assets/134455/14657714/58261828-065d-11e6-870f-352f41bbd8ee.png)